### DISCUSSION TITLE
Use WID

### DISCUSSION THREAD

**Discussion by Foucdvd** [neos-sdi/adfsmfa#285] [Q&A] [answered]
Hi,
I start using adfsmfa.
To store the information of users, i want used WID (Windows Internal Database) installed on the server ADFS.
Create MFADatabase : OK
first creation otp : OK

When i reboot server ADFS or restart service WID, the database MFADatabase is in state Recovery Pending. The other databases ADFS are OK.

I drop the MFADatase et recreate. reboot server ADFS or restart service WID . Same issue

Did you have this probleme ?

Thanks

**Comment by Foucdvd** [comment]
Informations supplémentaires : 
Version ADFS : 10.0.0.0

![Capture](https://user-images.githubusercontent.com/126770879/222406315-9e0d698a-a534-47a8-bc9b-6c1754cc982e.PNG)

**Comment by Foucdvd** [comment]
I don't understand the error with the WID base. 

Create the MFADatabase with the mmc : OK
Stop/start WID  MFADatabase :  KO
Drop MFADatabase : OK
Create the MFADatabase with the mmc : OK
Drop 3 tables (REGISTRATIONS, KEYDESCS and KEYS: OK
Stop/start WID  MFADatabase :  OK
Create table REGISTRATIONS : OK
Stop/start WID  MFADatabase :  KO

So i will try with the storage ADDS

**Comment by redhook62** [comment]
Hi,

Creating the MFA Database in WID is not supported, you MUST have a full version of SQLServer.

regards

**Comment by Foucdvd** [comment]
Hi redhook62,
OK Thanks. I understand now.

On my server ADFS, the Adfs service account is a service account GMSA.
Is it compatible with ADDS Storage ?

Thanks

**Comment by redhook62** [comment, marked as answer]
Yes, it's compatible and recommended for ADFS, To access your ADDS, you use the superaccount (everything is detailled in the wiki), because accessing ADDS with the Web Pool Account is a security risk.

**Comment by Foucdvd** [comment]
Hi,
With SQLEXPRES 2019 , all works.
Thanks
